### DISCUSSION TITLE
775

### DISCUSSION THREAD

**Discussion by giscus** [jojoldu/blog-comments#947] [Comments]
# 775

2024.02.12에 블로그의 누적 조회수가 1,000만이 되었다. 전체글은 585개가 되었다. 2015.1.31에 첫 글을 작성했으니, 약 9년만에 1,000만이 되었다. 예전에는 6개월에 100만씩 증가했다면, 요즘은 7개월에 100만씩 증가하고 있다. 아무래도 기술 문제를 해결한 사례보다는 현재의 내 생각을 계속 정리하다보니 구글 검색을 통해 유입되는 비율이 낮아진 것 같다. 1,000만이 오면서까지 있었던 일들을 연도별로 정리해보았다. 2015년 2015년 1월 31일에 첫 글을 작성했다. 누가봐도 성의없는 글이다. 처음 시작은 아웃사이더님, 권남님 의 블로그/위키를 보고 나도 이렇게 되고 싶다는 마음에 시작했다. 당시에도 여러 블로그 플랫폼이 있었고, 특히 개발자는 이글루스를 많이들 사용하고 있..

https://jojoldu.tistory.com/775

**Comment by 62hoon99** [comment]
좋은 글 감사합니다.

**Reply by jojoldu** [reply]
감사합니다! ㅎㅎ

**Comment by BenjaminKim** [comment]
[다크 패턴 이야기](https://jeho.page/essay/2024/03/19/resume-github-blog.html)가 또 나왔네요. 😢
*취업을 위한 블로그는 다크 패턴이다*가 아니라...
다크 패턴으로 블로깅하는 건 취업이든 스스로에게든 도움이 안된다는 내용입니다.
본문에 향로님 글을 복사해서 자기 블로그에 적었다는 이야기가 바로 **다크** 패턴이라 할 수 있겠네요.
매일 꾸준히 걸어가는 일 만큼 대단한 일도 없겠죠.

1000만 블로그 축하드립니다.

**Reply by jojoldu** [reply]
아고 @BenjaminKim 님 죄송합니다 ㅠㅠ 
본의아니게 재호님의 글을 저격하는 것처럼 되버렸네요.

원래 목적은 "프로세스 해킹처럼 취업하기 좋은 전략으로서 블로그는 확실히 메리트가 없다" 이게 핵심이였습니다.
재호님의 글을 인용한 트위터의 다크패턴 이야기를 보고 작성한 것이였는데, 재호님까지 심란하게 만들어드린것 같네요 ㅠ 
죄송합니다 ㅠ 

> 재호님의 글에서도 재호님이 다크패턴이다라고 이야기한 것이 아니라, **지인을 통해 그런 얘기를 들은것을 소개했다고** 저는 이해했습니다.

저도 항상 재호님이 꾸준히 공유해주시는 블로그 글 보면서 많이 예전 생각도 나고 지금의 고민도 많이 하는것 같아요 :) 
항상 좋은 글 써주셔서 고맙습니다!

**Comment by Hoo93** [comment]
항상 너무 잘 읽고 있습니다.
이제는 워낙 유명하시기에 처음에는 어떻게 시작하셨는지 예전 글을 찾아보기도 했었는데
동욱님이 정리해 주신 블로그 발전사를 보니까 너무 재밌었습니다.
감사합니다 !

저도 동욱님처럼 포기하지 않고 블로그 활동 이어가 보겠습니다

**Comment by Hoo93** [comment]
그리고 무엇보다 1000만 블로그 축하드립니다 !
